datadog_checks_dev/datadog_checks/dev/tooling/e2e/run.py

```diff
@@ -30,9 +32,9 @@ def _execute(check, command, env_vars):
 def start_environment(check, env):
     env_vars = {E2E_TEAR_DOWN: 'false', 'PYTEST_ADDOPTS': '--exitfirst'}
     if file_exists(get_hatch_file(check)):
-        command = f'hatch env run --env {env} test'
+        command = [sys.executable, "-m", 'hatch', 'env', 'run', '--env', env, 'test']
```

Comment:
nit - you're mixing `"` and `'` in the same line (told you it's a nit right? 😅 )

Reply:
Haha, yea, I'm so used to `black` fixing that for me (literally in every other project I've worked on), that I'm very sloppy about it myself. I guess in one line is too much though, I'll fix it along with the other thing.